Documentation/README_TapToPlace.md

```diff
@@ -23,7 +23,8 @@ Tap to Place is a far interaction component that is used to place a game object
         - `Auto Start`: If true, the Tap to Place solver will start out controlling the position of the game object to be placed. The object will immediately start following the TrackedTargetType (Head or Controller Ray). This value must be modified before Start() is invoked in order to have any effect.
         - `Default Placement Distance`: The default distance (in meters) an object will be placed relative to the TrackedTargetType forward in the SolverHandler. The game object will be placed at the default placement distance if a surface is not hit by the raycast.
         - `Max Raycast Distance`: The max distance (meters) for the raycast based on the 'TrackedTargetType' origin. This raycast looks for a surface to place a selected object.
-        - `Surface Normal Offset`: The distance between the center of the game object to place and a surface along the surface normal, if the raycast hits a surface. By default, the surface normal offset is 0 but after Start() the surface normal offset is set to the collider's extents along the z axis.
+        - `UseDefaultSurfaceNormalOffset`: This property is true by default, it ensures the object being placed is aligned on a surface. If this property is true, the default surface normal offset will be applied instead of any value specified for the `SurfaceNormalOffset` property. If false, the value for `SurfaceNormalOffset` will be applied. The default surface normal offset is the collider's extents along the z axis.
+        - `Surface Normal Offset`: The distance between the center of the game object to place and a surface along the surface normal, if the raycast hits a surface. 
```

Comment:
Even though this would be double mentioning, it's probably still worth adding the note that this is only honored if UseDefaultSurfaceNormalOffset is false.